src/connectors/range-slider/connectRangeSlider.js

```diff
@@ -76,108 +80,146 @@ export default function connectRangeSlider(renderFn) {
     };
 
     return {
-      getConfiguration: originalConf => {
-        const conf = {
-          disjunctiveFacets: [attributeName],
-        };
+      _isAbleToRefine(min, max) {
+        const isMinValid = _isFinite(min);
```

Comment:
`_isAbleToRefine` is called once with minBounds and maxBounds as the values, right? If yes this seems superfluous.